models/shopify.yml

```diff
@@ -721,4 +721,106 @@ models:
+        description: Boolean representing whether the generated discount code will be valid only for a single use per customer. This is tracked using customer ID.
+      - name: prereq_min_quantity
+        description: If `customer_selection` is `prerequisite`, the minimum number of items for the price rule to be applicable. The quantity of an entitled cart item must be greater than or equal to this value.
+      - name: prereq_max_shipping_price
+        description: If `customer_selection` is `prerequisite`, the maximum shipping price for the price rule to be applicable. The shipping price must be less than or equal to this value
+      - name: prereq_min_subtotal
+        description: If `customer_selection` is `prerequisite`, the minimum subtotal for the price rule to be applicable. The subtotal of the entitled cart items must be greater than or equal to this value for the discount to apply.
+      - name: prereq_min_purchase_quantity_for_entitlement
+        description: If `customer_selection` is `prerequisite`, the prerequisite purchase for a Buy X Get Y discount. The minimum purchase amount required to be entitled to the discount.
+      - name: prereq_buy_x_get_this
+        description: If `customer_selection` is `prerequisite`, in a Buy/Get ratio for a Buy X Get Y discount, this is the offered 'get' quantity.
+      - name: prereq_buy_this_get_y
+        description: If `customer_selection` is `prerequisite`, in a Buy/Get ratio for a Buy X Get Y discount, this defines the necessary 'buy' quantity.
+      - name: starts_at
+        description: The date and time (ISO 8601 format) when the price rule starts.
+      - name: target_selection
+        description: >
+          The target selection method of the price rule. Valid values include `all` (the price rule applies the discount to all line items in the checkout) and 
+          `entitled` (the price rule applies the discount to selected entitlements only).
+      - name: target_type
+        description: The target type that the price rule applies to. Valid values include `line_item` (the price rule applies to the cart's line items) and `shipping_line` (the price rule applies to the cart's shipping lines).
+      - name: title
+        description: >
+          The title of the price rule. This is used by the Shopify admin search to retrieve discounts. It is also displayed on the Discounts page of the Shopify admin for bulk discounts. 
+          Shopify recommends that this map onto the associated `discount_code.code`.
+      - name: pirce_rule_updated_at
+        description: The date and time (ISO 8601 format) when the price rule was updated.
+      - name: usage_limit
+        description: The maximum number of times the price rule can be used, per discount code.
+      - name: value
+        description: The value of the price rule. If if the value of `target_type` is `shipping_line`, then only -100 is accepted. The value must be negative.
+      - name: value_type
```

Comment:
yeah -- would that be better to test here or upstream in the staging model?

similarly, i wonder if we should do that for fulfillment status as well because of https://github.com/fivetran/dbt_shopify/blob/feature/package-revamp/models/shopify__inventory_levels.sql#L117

Reply:
Good call, I think applying the test at the staging model would make more sense and is in line with our current practices.

Also, yeah great catch - let's apply the same warning test to the fulfillment status.